vignettes/data_standardization.Rmd

```diff
@@ -31,7 +31,7 @@ options(scipen=999)
 Formatting and data descriptions for standardized datasets.
```

Comment:
There is too much going on in this markdown. I think it would be great if you could separate this into a separate markedown for each data type. That way it will be easier to just look at prey density which is the main focus here.

Reply:
Please make that update and then I will do another review. It would be great if these were github compatible markdowns

Reply:
Would be nice to then have a general methods vignette that links to all of these standardization pieces.